rust/operator-binary/src/restart_controller/pod.rs

```diff
@@ -81,14 +81,22 @@ pub async fn start(client: &Client, watch_namespace: &WatchNamespace) {
 }
 
 async fn reconcile(pod: Arc<Pod>, ctx: Arc<Ctx>) -> Result<Action, Error> {
+    tracing::info!("Starting reconciliation ..");
     if pod.metadata.deletion_timestamp.is_some() {
         // Object is already being deleted, no point trying again
+        tracing::info!("Pod is already being deleted, taking no action!");
         return Ok(Action::await_change());
     }
 
-    let pod_expires_at = pod
-        .metadata
-        .annotations
+    let annotations = &pod.metadata.annotations;
+
+    tracing::debug!("Found expiry annotations: [{:?}]", annotations);
```

Comment:
```suggestion
    tracing::debug!(pod.annotations = ?annotations, "Found expiry annotations");
```

Reply:
That said, this is _all_ the annotations period, would it make sense to restrict it to restart annotations?